README.md

```diff
@@ -14,7 +14,7 @@ Prior to executing the tests ensure you have:
 Run the following command to start services locally:
 
     sudo mongod
-    sm --start UI_TEST_TEMPLATE -f
+    sm --start UI_TEST_TEMPLATE -r
```

Comment:
UI_TEST_TEMPLATE ?   What services does that start?

Reply:
`sm --start NDRC_ALL -r` perhaps?

Reply:
Dunno - Hmm - it looks like the boilerplate name for the generated template.  I find it in a lot of README's for other services along with <digital service name>.  Guess this README has not been fully updated - let me do that now.